digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://ifix.digit.org/platform/strategy-and-approach/functional-specifications",
+      "https://ifix.digit.org/platform/strategy-and-approach/technical-specification"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Dashboards and data dump downloads from dashboards in XLS format."
```

Comment:
XLS is a Microsoft proprietary Binary Interchange File Format. Please check with the submitter whether CSV/ XML/ JSON or any other non-proprietary formats are supported via the data dump downloads from the dashboards. Thanks.

Reply:
Since iFix uses PostgreSQL as the datastore and it is open source, there are standard
tools and processes available to extract data from the database in non-proprietary
formats - JSON, CSV etc.